src/index.tsx

```diff
@@ -359,9 +360,23 @@ const SnackbarPopup = styled(motion.div)`
   }
 `
 
+let browserIsOld = false
+
+try {
+  BigInt(1)
+} catch (e) {
+  browserIsOld = true
```

Comment:
Nice one!

Reply:
No need to define e as a param though (eslint wouldn't be happy I believe?)

Reply:
yep, you believe correct.

![image](https://user-images.githubusercontent.com/1579899/151000799-38fd5fe0-2907-44b4-96c5-5cbf8aea4ea7.png)